.flowconfig

```diff
@@ -8,6 +8,7 @@
 .*/node_modules/htmltojsx/.*
 .*/node_modules/documentation/.*
 .*/node_modules/module-deps/.*
+.*/node_modules/bcryptjs/src/bower.json
```

Comment:
What's the rationale for this change? Doesn't look like bcryptjs is in the dependency tree.

Reply:
I ran into something similar to this the other day when I was using `npm install` rather than `yarn install`. I can't say for sure that's where this was introduced but I kept getting a test error in bcrypt until I removed node modules and reinstalled with yarn. This shouldn't be needed.

Reply:
It's present in my node_modules after a fresh `npm i` and causes flow test failures thanks to [some templating wackiness](https://github.com/dcodeIO/bcrypt.js/blob/684fac6814a81d974c805a15e22fd69922c7ca6e/src/bower.json.template#L4). Not related to this PR and I'm happy to do whatever's most appropriate here. Replicated with node 6 and 8.

Reply:
Tom, if you remove this line from `.flowconfig` then `rm -rf node_modules` and reinstall with `yarn install` does the bcrypt error go away?

Reply:
`npm install` doesn't use `yarn.lock`, so it installs a newer version of `@mapbox/batfish`, which pulls in bcryptjs via some transitive dependencies.

Let's revert here and consider using https://www.npmjs.com/package/use-yarn to avoid this gotcha in the future.

Reply:
@ryanhamley yes, confirmed that using yarn avoids the problem

@jfirebaugh sounds good, I've removed the relevant commit

Reply:
I opened a ticket for the npm bug https://github.com/mapbox/mapbox-gl-js/issues/6774